docs/extensions/accounts/tests.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Tests"
+weight: 80
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/extensions/accounts
+geekdocFilePath: tests.md
+---
+
+{{< toc >}}
+
+## Requirements
+
+You need a working installation of [the Go programming language](https://golang.org/), [the Node runtime](https://nodejs.org/) and [the Yarn package manager](https://yarnpkg.com/) installed to run the acceptance tests. You may also want to use [Docker](https://www.docker.com/) to start the necessary services in their respective containers.
+
+## Acceptance Tests
+
+Make sure you've cloned the [web frontend repo](https://github.com/owncloud/web/) and the [infinite scale repo](https://github.com/owncloud/ocis/) next to each other. If your file/folder structure is different, you'll have to change the paths below accordingly.
+
+### In the web repo
+
+Install dependencies and build the frontend with a watcher by running
+
+```
+yarn && yarn build:w
+```
+
+Also, start the necessary services. The easiest way to do this is using Docker (Compose):
+
+```
+docker compose up selenium middleware-ocis vnc ocis
+```
+
+### In the oCIS repo
+
+Navigate into the accounts service via `cd accounts/` and start the acceptance tests by running
```

Comment:
So we're testing the accounts service of the ocis latest master here, not the one from a branch the respective dev has been working on.

Reply:
I think the best approach would be to not start the `ocis` container from the web repo (then you can also skip the `yarn build` command for web), but only selenium, middleware-ocis and vnc. Then in ocis, build the binary (including the changes in the accounts ui) and run the tests like you described it already.

Reply:
aaaaaah wait a minute. There are two different use cases for testing here:
1) I'm upgrading to the most recent web release and want to check if the accounts ui doesn't break with it
2) I'm modifying the accounts ui and want to check if my modifications don't break the accounts ui (without caring about different web assets)

I think that should be explicitly stated and described here in the docs. :-) Sorry that I didn't recognize that before.

Reply:
I've tried to make the possibilities more obvious, please re-review :) and thanks @wkloucek for unblocking!